fsw/cfe-core/src/inc/cfe_error.h

```diff
@@ -42,6 +42,19 @@
 /* Include Files */
 #include "osapi.h"
 
+/*
+ * Define a type for readability.
+ */
+
+typedef int32 CFE_Status_t;
+typedef CFE_Status_t CFE_RC_t;
+
+/*
+ * Utility macros.
+ */
+#define CFE_ISSUCCESS(S) ((S) & CFE_SEVERITY_BITMASK != CFE_SEVERITY_ERROR)
+#define CFE_ISINFO(S) ((S) & CFE_SEVERITY_BITMASK == CFE_SEVERITY_INFO)
```

Comment:
can these be inline functions instead?  These are just as fast but more type safe and don't have the same risks/issues as macros.

Reply:
Just as fast if inlining is enabled. If it's not, and these are "real" functions, there's a decent amount of overhead.

If they are declared inline, they should be `static inline` at least, to avoid GOT/PLT lookup and such.